### ISSUE TITLE
Richtext: images should have caption and alt text(richtext)

### ISSUE THREAD

**Issue by barborapuchlova** [bratislava/olo.sk#698]
The key part of code: https://github.com/bratislava/olo.sk/blob/3e812e877c2d926448bcddfd3ad44faacb5a0577/strapi/src/plugins/wysiwyg/admin/src/components/Wysiwyg/index.tsx#L50

Add caption.

Display caption on FE and use alt text if not used on FE.
